ui/packages/catalog-ui-search/src/main/webapp/component/content/content.view.js

```diff
@@ -33,8 +33,62 @@ const QueryTitleView = require('../query-title/query-title.view.js')
 const GoldenLayoutView = require('../golden-layout/golden-layout.view.js')
 const LoadingCompanionView = require('../loading-companion/loading-companion.view.js')
 
+import { ChangeBackground } from '../../react-component/styles/mixins/change-background'
+import MultiSelectActions from '../../react-component/container/multi-select-actions'
+import styled from '../../react-component/styles/styled-components'
+
+const ContentLeft = styled.div`
+  ${props => {
```

Comment:
❓ Just curious: does this return css elements given the passed in `props`?

Reply:
yep, it's a mixin: https://github.com/codice/ddf/blob/master/ui/packages/catalog-ui-search/src/main/webapp/react-component/styles/mixins/change-background.js